adapter/src/main/java/edu/artic/adapter/BaseRecyclerViewAdapter.kt

```diff
@@ -37,12 +39,11 @@ abstract class BaseRecyclerViewAdapter<TModel, VH : BaseViewHolder>(
     interface OnItemClickListener<in T> {
 
         /**
-         * The item clicked.
-
-         * @param position   The position in the itemslist that was clicked.
-         * *
+         * Callback for [on-click listeners][View.setOnClickListener]. Always invoked
+         * on main thread.
```

Comment:
"Always invoked on main thread."
Isn't this always a given?

Reply:
Honestly, I was trying to follow some of the framework docs - they'll include a small sentence like that on listener interfaces.